package.json

```diff
@@ -48,6 +48,6 @@
     "opal-compiler": "1.0.7"
   },
   "peerDependencies": {
-    "asciidoctor.js": ">=1.5.7-rc.1 <1.6"
+    "@asciidoctor/core": ">=2.0.0-rc.1 <3"
```

Comment:
I've been playing with https://semver.npmjs.com/ and as far as I know `^2.0.0-rc.1` produces the same result as `>=2.0.0-rc.1 <3`. I might be wrong though and I don't really know which one is the "best"... but if possible we should use the same version.

Reply:
Nice! I thought the prerelease bit was not included in the caret. It makes things simpler, thank you 🙂